tensorflow/tools/docker/Dockerfile.gpu

```diff
@@ -58,6 +58,9 @@ COPY notebooks /notebooks
 # We just add a little wrapper script.
 COPY run_jupyter.sh /
 
+# We depend on libcupti.so
+ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
```

Comment:
Yea, it's installed, when you manually change the environment variable, it works. 
See #7207